tests/unit/records/test_records_receivers.py

```diff
@@ -541,7 +542,8 @@ def test_populate_experiment_suggest_populates_if_record_is_experiment():
     assert json_dict['experiment_suggest']['input'] == \
         ['foo', 'bar', 'foo_var', 'bar_var']
     assert json_dict['experiment_suggest']['output'] == 'foo'
-    assert json_dict['experiment_suggest']['payload']['$ref'] == 'http://foo/$ref'
+    assert json_dict['experiment_suggest'][
+        'payload']['$ref'] == 'http://foo/$ref'
```

Comment:
do we have formatting script or something? to run before because usually when this always happen to me when I format with my text editor.